modules/ocf_docker/files/garbage-collect

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+# Always make sure the registry is started, no matter what fails.
+trap 'systemctl start docker-registry' EXIT
```

Comment:
Nice, this is something I've never actually used in bash, but have seen around. This is a pretty good use of it :)